app/constants/defaultRuleContent.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_RULE_CONTENT = { nodes: [], edges: [] };
```

Comment:
Encountering an issue with uploading rules created on the platform, where the jdm editor rejects json that is not coded with the right content type. I think adding contentType: "application/vnd.gorules.decision" to the default rule content, before nodes and edges, should fix this issue?

Reply:
Oh, I didn't test uploading. That's good to know. I should fix this right away.